starters/next-drupal-starter/pages/examples/ssg-isr.jsx

```diff
@@ -0,0 +1,101 @@
+import { NextSeo } from "next-seo";
+import { isMultiLanguage } from "../../lib/isMultiLanguage";
+import {
+  getCurrentLocaleStore,
+  globalDrupalStateStores,
+} from "../../lib/drupalStateContext";
+
+import { ArticleGridItem, withGrid } from "../../components/grid";
+import Layout from "../../components/layout";
+
+export default function HomepageTemplate({
```

Comment:
Update name for this template
```suggestion
export default function SSGISRExampleTemplate({
```